1-exercises/A-reading-list/script.js

```diff
@@ -25,4 +21,36 @@ const books = [
   },
 ];
 
-readingList(books);
+function readingList(books) {
+  // Write your code here...
+//   const contentDiv = document.getElementById("content");
+  const ul = document.getElementById("reading-list");
+//   const ul = document.createElement("ul");
+
+  books.forEach((book) => {
+    const li = document.createElement("li");
+    const p = document.createElement("p");
+    const img = document.createElement("img");
+
+    p.textContent = `${book.title} + ${book.author}`;
+    img.src = book.bookCoverImage;
+
+    li.appendChild(p);
+    li.appendChild(img);
+    ul.appendChild(li);
+
+    if (book.alreadyRead) {
+      li.classList.add("read");
+      li.style.backgroundColor = "green";
+    
+    } else { 
+      li.classList.add("unread");
+      li.style.backgroundColor = "red";
+    }
```

Comment:
```javascript
li.classList.add(book.alreadyRead ? 'read' : 'unread'  )
```

Reply:
Thank you for add,  ternary operator has been modified